framework/src/main/java/org/checkerframework/common/value/PropertyFileHandler.java

```diff
@@ -0,0 +1,319 @@
+package org.checkerframework.common.value;
```

Comment:
Is it worth moving the classes for the property file handler into a separate package?

Reply:
For now, I think we should keep this in the same package as the value checker. We can do the refactoring when we have other handlers.